doc/source/release-notes/Percona-Server-5.6.41-84.1.rst

```diff
@@ -0,0 +1,38 @@
+.. rn:: 5.6.41-84.1
+
+================================================================================
+|Percona Server| 5.6.41-84.1
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| 5.6.41-84.1 on
+August 1, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.41-84.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+This release merges changes of `MySQL 5.6.41
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-41.html>`_, including
+all the bug fixes in it. Percona Server for MySQL 5.6.41-84.1 is now the current
+GA release in the 5.6 series. All of Percona’s software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- A simple SELECT query on a table with CHARSET=euckr COLLATE=euckr_bin could
+  return different results each time it was executed. Bug fixed :psbug:`4513`
+  (upstream :mysqlbug:`91091`).
+- Percona Server 5.6.39-83.1 crashed when altering a table. Bug fixed
```

Comment:
could crash when altering an InnoDB table that had a full-text search index defined